### ISSUE TITLE
No carga el mapa

### ISSUE THREAD

**Issue by denissoto** [datosgobar/densidad-poblacion#5]
* ¿Cuál es el comportamiento que esperabas y cuál fue el comportamiento recibido? 
Esperaba que cargue el mapa al acceder al sitio.

* ¿Cuáles son los pasos para reproducir el bug?
Acceder al sitio

* Detalles del contexto: ¿qué sistema operativo, qué versión de navegador estás usando, por ejemplo?
Windows 10, 2004
Chrome 84.0.4147.135

* ¿Tenés algún stack trace o captura de pantalla? Adjuntalo.  

![image](https://user-images.githubusercontent.com/24607913/91004675-40598b80-e5ab-11ea-842f-b23cc4202ccd.png)

**Comment by Salo89** [comment]
Mismo problema por acá. Pudiste solucionarlo?
Hay forma de montar el server en mi PC aunque sea?
gracias

**Comment by denissoto** [comment]
> Mismo problema por acá. Pudiste solucionarlo?
> Hay forma de montar el server en mi PC aunque sea?
> gracias

No, no pude solucionarlo. Y tampoco sé si lo puedo montar en mi PC.

**Comment by Salo89** [comment]
Yo instale el plugin de http-server para node.js (https://jasonwatmore.com/post/2016/06/22/nodejs-setup-simple-http-server-local-web-server) , "levante" el server, pero no encuentro forma de que funcione.

**Comment by lsalomon-uala** [comment]
Para entra a la pagina se puede hacer por acá: https://datosgobar.github.io/densidad-poblacion/
No carga el mapa porque en un momento quiere obtenerlo desde una api de "mapbox". Devuelve unauthorized, así que esperen sentados porque eso no va a andar ya que sospecho que el token venció y nadie lo actualizó ya que en el codigo está super hardcodeado.

/public/javascript/es5/app.js
`function renderMap(data, varGlobal) {
  139:         // MapBox
  140:         mapboxgl.accessToken = 'pk.eyJ1IjoiZ29uemFsb2JiIiwiYSI6ImNqMXBjY3BsajAwMWgzM253ZnJwbjlxb2sifQ.FNY4jZ525MbOkxg0oVKljw';`

Request:
`https://api.mapbox.com/styles/v1/gonzalobb/cjz1ixb3w25991crtkt1n68zy?access_token=pk.eyJ1IjoiZ29uemFsb2JiIiwiYSI6ImNqMXBjY3BsajAwMWgzM253ZnJwbjlxb2sifQ.FNY4jZ525MbOkxg0oVKljw`

**Comment by cosentinowebdev** [comment]
### **lo estuve viendo y creo que este proyecto quedo viejo**
_que se puede hacer?_ 
ir a mapbox y crear uno nuevo 
_por que y no hacer ingeniería inversa?_
en la hora de mi vida que perdí en esto no encontré los estilos que definió el autor. 
_podría sacarlos?_ 
si
_lo hare ?_ 
no

**les dejo links que quizás a futuros programadores que visiten este repo les sirva** 
https://account.mapbox.com/auth/signup/
https://account.mapbox.com/access-tokens
https://studio.mapbox.com/

si alguien lo hace me avisa
